testing/integration/logs_ingestion_test.go

```diff
@@ -143,6 +211,7 @@ func testMonitoringLogsAreShipped(
 			"add_cloud_metadata: received error failed requesting openstack metadata: Get \\\"https://169.254.169.254/2009-04-04/meta-data/instance-type\\\": dial tcp 169.254.169.254:443: connect: connection refused",               // okay for the cloud metadata to not work
 			"add_cloud_metadata: received error failed with http status code 404", // okay for the cloud metadata to not work
 			"add_cloud_metadata: received error failed fetching EC2 Identity Document: operation error ec2imds: GetInstanceIdentityDocument, http response error StatusCode: 404, request to EC2 IMDS failed", // okay for the cloud metadata to not work
+			"failed to invoke rollback watcher: failed to start Upgrade Watcher: fork/exec /var/lib/elastic-agent/elastic-agent: no such file or directory",                                                   //on debian this happens probably need to fix.
```

Comment:
I assume this only happens for the DEB? If we are the DEB we probably shouldn't even be launching the watcher, or I suppose we could fix the path.

Either way this is worth a follow up issue.

Reply:
+1

Reply:
https://github.com/elastic/elastic-agent/issues/4332